synapse/handlers/admin.py

```diff
@@ -247,6 +248,21 @@ async def export_user_data(self, user_id: str, writer: "ExfiltrationWriter") ->
                 )
                 writer.write_state(room_id, event_id, state)
 
+        # Get the user profile
+        profile = await self.get_user(UserID.from_string(user_id))
+        assert profile
```

Comment:
I think previously if you gave a bogus user ID (`@does_not_exist:example.com`) you'd end up with a bunch of empty directories, now you'll get an assertion error? Or do we check somewhere that the user exists?

Regardless I assume this exists to make mypy happy?

```suggestion
        assert profile is not None
```

Reply:
Done